### DISCUSSION TITLE
rules provider功能的问题

### DISCUSSION THREAD

**Discussion by SekiBetu** [MetaCubeX/mihomo#90] [Q&A] [answered]
请问meta核中的rules provider功能与premium中的是一致的吗，还是自己模仿着开发的功能
如果是一致的，是要了原作者授权把他开源出来了吗？
很疑惑

**Comment by MetaCubeX** [comment, marked as answer]
惜别 ***@***.***>於 2022年6月14日 週二，02:11寫道：

> 请问meta核中的rules provider功能与premium中的是一致的吗，还是自己模仿着开发的功能
> 如果是一致的，是要了原作者授权把他开源出来了吗？
> 很疑惑
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MetaCubeX/Clash.Meta/discussions/90>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUDYGU5HH35EMMBD4VRKFRDVO52ULANCNFSM5YVB47RA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
自己开发出来的，闭源的怎么可能拿到代码
